tools/tidb-binlog-kafka.md

```diff
@@ -117,52 +127,71 @@ zookeeper_addrs = "192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181"
 
 ### Deploy Pump using Binary
 
+A usage example:
+
+For three PDs, three ZooKeepers, and one TiDB, the information of each node is as follows: 
```

Comment:
-> Assume that we have three PDs, three ZooKeepers, and one TiDB. The information of each node is as follows: 

(to keep it consistent with other similar examples)